src/providers/activeLearning/electronProxyHandler.test.ts

```diff
@@ -0,0 +1,33 @@
+jest.mock("../storage/localFileSystemProxy");
+import { LocalFileSystemProxy } from "../storage/localFileSystemProxy";
+import { ElectronProxyHandler } from "./electronProxyHandler";
+import * as tf from "@tensorflow/tfjs";
+// tslint:disable-next-line:no-var-requires
+const modelJson = require("../../../cocoSSDModel/model.json");
+
+describe("Load default model from filesystem with TF io.IOHandler", () => {
+    it("Check file system proxy is correctly called", async () => {
+        const storageProviderMock = LocalFileSystemProxy as jest.Mock<LocalFileSystemProxy>;
+        storageProviderMock.mockClear();
+
+        storageProviderMock.prototype.readText = jest.fn((fileName) => {
+            return JSON.stringify(modelJson);
+        });
+
+        storageProviderMock.prototype.readBinary = jest.fn((fileName) => {
+            return [];
+        });
+
+        const handler = new ElectronProxyHandler("folder");
+        try {
```

Comment:
Why empty catch block?  If the model fails it should bubble up or be handled.

Reply:
We are not really loading the model here.  Weights are not provided/mocked so loading the model here will fail.  As in the description of the test the goal here is to test the ElectronProxyHandler class that implement the tensorflow.js io.IOHandler interface.  In particular here we want to check the passing a filesystem path the internal TF api call our proxied readText and readBinary methods to access the filesystem from the browser.  The readText is mocked to return a real tf.js json model with link to 5 binary weights file.  The weight files are not mocked but the test verify that the readBinary is actually called 5 times.